coc/client.py

```diff
@@ -115,27 +102,19 @@ class Client:
 
     """
 
-    def __init__(self, tokens=None, loop=None, email=None, password=None, update_tokens=False):
-        self.loop = loop or asyncio.get_event_loop()
-        has_auth = password or email
-
-        if tokens:
-            if isinstance(tokens, str):
-                tokens = [tokens]
-            elif isinstance(tokens, list):
-                pass
-            else:
-                raise RuntimeError('tokens must be either a str or list of str tokens')
+    def __init__(self, email, password, key_count=1,
+                 key_names='Created with coc.py Client', loop=None):
 
-        else:
-            if not has_auth:
-                raise RuntimeError('An email and password must be set if no tokens are provided')
+        self.loop = loop or asyncio.get_event_loop()
+        correct_key_count = max(min(KEY_MAXIMUM, key_count), KEY_MINIMUM)
```

Comment:
does this line work? i havent tested but in my head it doesnt.....or at least what line 111 does below

Reply:
nvm i missed the `not` bit